util/test/tests/D3D11/D3D11_Shader_Debug_Zoo.py

```diff
@@ -38,6 +40,40 @@ def check_capture(self):
                 self.controller.FreeTrace(trace)
 
             rdtest.log.success("Test {} matched as expected".format(test))
+        rdtest.log.dedent()
+
+        rdtest.log.print("Performing MSAA tests:")
+        rdtest.log.indent()
+        draw = draw.next
+        self.controller.SetFrameEvent(draw.eventId, False)
+        pipe: rd.PipeState = self.controller.GetPipelineState()
+        for test in range(4):
+            # Debug the shader
+            trace: rd.ShaderDebugTrace = self.controller.DebugPixel(4, 4, test,
+                                                                    rd.ReplayController.NoPreference)
+
+            # Validate that the correct sample index was debugged
+            inputs: List[rd.ShaderVariable] = list(trace.inputs)
```

Comment:
Was this necessary to work around a problem? `trace.inputs` should be usable anywhere a list is via python duck typing. Aside from debugging visualisation I've found only works on true lists.